packages/hub/node-tests/routes/prepaid-card-customizations-test.ts

```diff
@@ -0,0 +1,277 @@
+import { Server } from 'http';
+import supertest, { Test } from 'supertest';
+import { bootServerForTesting } from '../../main';
+import { Client } from 'pg';
+import shortUuid from 'short-uuid';
+import { Container, Registry } from '../../di/dependency-injection';
+import { parseIdentifier } from '@cardstack/did-resolver';
+
+let handleValidateAuthToken = function (_encryptedAuthToken: string) {
+  return '';
+};
+
+const stubNonce = 'abc:123';
+let stubAuthToken = 'def--456';
+let stubTimestamp = process.hrtime.bigint();
+
+class StubAuthenticationUtils {
+  generateNonce() {
+    return stubNonce;
+  }
+  buildAuthToken() {
+    return stubAuthToken;
+  }
+  extractVerifiedTimestamp(_nonce: string) {
+    return stubTimestamp;
+  }
+
+  validateAuthToken(encryptedAuthToken: string) {
+    return handleValidateAuthToken(encryptedAuthToken);
+  }
+}
+
+let stubUserAddress = '0x2f58630CA445Ab1a6DE2Bb9892AA2e1d60876C13';
+handleValidateAuthToken = function (encryptedString: string) {
```

Comment:
Can this be assigned at the declaration of `handleValidateAuthToken`, since we use the same definition throughout?

Reply:
Yes. Copy/paste oversight.

Reply:
Fixed ✅